unistd/fts.c

```diff
@@ -0,0 +1,22 @@
+// fts.c
+// Provided by brimdata on 2022/10/19
+// Copyright (c) 2019 Robin.Rowe@CinePaint.org
```

Comment:
This and a few other files had this copyright header already in place when we got them from Microsoft, but they weren't already in the repo. We're fine with assigning the copyright to you here, but I just want to bring attention to it.

Reply:
You can't assign copyright if you're not the owner.

Reply:
I got ahold of Noah Treuhaft at brimdata, who was the original source of the files. I've fixed all of the copyright attributions for those files to match the requested format.